Makefile

```diff
@@ -85,6 +85,10 @@ manifests: bin/kustomize generate
 	kustomize build config/certmanager > charts/conduit-operator/templates/certificate.yaml
 	kustomize build config/webhook > charts/conduit-operator/templates/webhook.yaml
 
+.PHONY: install
+install: manifests bin/kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
```

Comment:
best to have these installed when the operator boots up, as it will register them automatically. 
otherwise there may be discrepancy between crd and code handling the new stuff there.